deploy/olm-catalog/bundle/manifests/oadp-operator.v0.1.3.clusterserviceversion.yaml

```diff
@@ -211,7 +210,7 @@ spec:
                 - name: VELERO_OPENSHIFT_PLUGIN_REPO
                   value: openshift-velero-plugin
                 - name: VELERO_OPENSHIFT_PLUGIN_TAG
-                  value: unified-repo
+                  value: latest
```

Comment:
Should we not change this in the `operator.yaml` as well ?

Reply:
Oh yes thanks!